doc/websocket.qbk

```diff
@@ -67,43 +69,47 @@ both Boost.Asio and the WebSocket protocol specification described in
 [section:creating Creating the socket]
 
 The interface to Beast's WebSocket implementation is a single template
-class [link beast.ref.websocket__stream `websocket::stream`] which wraps a
-"next layer" object. The next layer object must meet the requirements of
-`SyncReadStream` and `SyncWriteStream` if synchronous operations are performed,
-`AsyncReadStream` and `AsyncWriteStream` is asynchronous operations are
-performed, or both. Arguments supplied during construction are passed to
-next layer's constructor. Here we declare two websockets which have ownership
-of the next layer:
+class [link beast.ref.websocket__stream `beast::websocket::stream`] which
+wraps a "next layer" object. The next layer object must meet the requirements
+of [link beast.types.streams.SyncStream [*`SyncReadStream`]] if synchronous
+operations are performed, or
+[link beast.types.streams.AsyncStream [*`AsyncStream`]] if asynchronous
+operations are performed, or both. Arguments supplied during construction are
+passed to next layer's constructor. Here we declare two websockets which have
+ownership of the next layer:
 ```
-io_service ios;
-websocket::stream<ip::tcp::socket> ws(ios);
+boost::asio::io_service ios;
+beast::websocket::stream<boost::asio::ip::tcp::socket> ws(ios);
 
-ssl::context ctx(ssl::context::sslv23);
-websocket::stream<ssl::stream<ip::tcp::socket>> wss(ios, ctx);
+boost::asio::ssl::context ctx(boost::asio::ssl::context::sslv23);
+beast::websocket::stream<
+    boost::asio::ssl::stream<boost::asio::ip::tcp::socket>> wss(ios, ctx);
 ```
 
 For servers that can handshake in multiple protocols, it may be desired
 to wrap an object that already exists. This socket can be moved in:
```

Comment:
You use "object" and "socket" to refer to, seemingly, the same thing. This can be a bit confusing. Might make more sense if you did this:

> For servers that can handshake in multiple protocols, it may be desired to wrap an object that already exists. This can be achieved by moving the object in. For example, a socket could be moved in: